interfaces/builtin/steam_support.go

```diff
@@ -138,8 +138,8 @@ mount options=(rw, rbind) /oldroot/home/{,**} -> /newroot/home/{,**},
 mount options=(rw, rbind) /oldroot/snap/** -> /newroot/snap/**,
 mount options=(rw, rbind) /oldroot/home/**/usr/ -> /newroot/usr/,
 mount options=(rw, rbind) /oldroot/home/**/usr/etc/** -> /newroot/etc/**,
-mount options=(rw, rbind) /oldroot/home/**/usr/etc/ld.so.cache -> /newroot/run/pressure-vessel/ldso/runtime-ld.so.cache,
-mount options=(rw, rbind) /oldroot/home/**/usr/etc/ld.so.conf -> /newroot/run/pressure-vessel/ldso/runtime-ld.so.conf,
+mount options=(rw, rbind) /oldroot/home/**/usr/etc/ld.so.cache -> /newroot/*/pressure-vessel/ldso/runtime-ld.so.cache,
+mount options=(rw, rbind) /oldroot/home/**/usr/etc/ld.so.conf -> /newroot/*/pressure-vessel/ldso/runtime-ld.so.conf,
```

Comment:
At the risk of repeating myself, these paths are an implementation detail. It is very likely that we will find that we need to do something slightly different in a future release.

For instance, if we find that we also need to bind-mount `ld.so.conf.d` because otherwise the container runtime framework doesn't work on, I don't know, Gentoo or something, then we will have to start doing that; and if snapd doesn't like that, then Steam will just not work.

Reply:
I think @smcv is right, we've seen paths like this change it would be great to make them as general as possible.  I'd prefer `/newroot/**,`

Reply:
Could we rely on the `pressure-vessel` directory, so, something like `/newroot/*/pressure-vessel/**,`? If `/newroot/**,` is the solution here, was there a reason that the initial rules were so restrictive (maybe @kenvandine knows that context, I'm not sure)?

Reply:
We usually lean more towards restrictive rules, so specifying more of the path is a common pattern.  However, steam-support is more permissive than most interfaces, so it would be good to accept the nature of that, IMO.  We'll have to get a security review anyway, so I'd recommend starting there and we can tighten it down more if necessary.

Reply:
I've changed each pressure-vessel based path to `/newroot/**,`

Reply:
> I've changed each pressure-vessel based path to `/newroot/**,`

Thanks, what you have done for e.g. the Wayland and X11 sockets looks like a big improvement.

It seems a little ironic that as far as I can see, you haven't done this for the line I started this thread from? :-)

Reply:
> It seems a little ironic that as far as I can see, you haven't done this for the line I started this thread from? :-)

Silly mistake, fixed now :)